wormhole-connect/src/config/index.ts

```diff
@@ -244,6 +228,8 @@ export function setConfig(customConfig?: WormholeConnectConfig) {
     /* @ts-ignore */
     config[key] = newConfig[key];
   }
+  /* @ts-ignore */
+  window.config = config;
```

Comment:
Should we call this something more specific so we don't overwrite `config` if it already exists? It's a pretty generic name. Also, is this used outside of debugging purposes?

Reply:
it's only for debugging. i can make it more obscure